UPGRADE-2.7.md

```diff
@@ -525,6 +525,14 @@ PropertyAccess
 Config
 ------
 
- * The `__toString()` method of the `\Symfony\Component\Config\ConfigCache` is marked as 
+ * The `__toString()` method of the `\Symfony\Component\Config\ConfigCache` is marked as
    deprecated in favor of the new `getPath()` method.
-   
+
+Yaml
```

Comment:
Ha ok. I didn't understand why only fappot writte the CHANGELOG so I didn't touch it. I didn't know there were CHANGELOG for each component.